src/Operations.jl

```diff
@@ -2214,6 +2214,9 @@ function status_ext_info(pkg::PackageSpec, env::EnvCache)
             # Check if deps are loaded
             extdeps_info= Tuple{String, Bool}[]
             for extdep in extdeps
+                haskey(weakdepses, extdep) ||
+                    pkgerror(isnothing(pkg.name) ? "M" : "$(pkg.name) has a m",
```

Comment:
I searched through Pkg for "malformed" and didn't find anything. String line-breaking in the middle of a word is a bit much, no? 😆

Reply:
Oh I feel for you, I'm more than familiar with similar situations myself. We could switch this to apply `uppercasefirst` to `"malformed"` conditionally if `isnothing(pkg.name)`?